docs/ssm-deployment-guide.md

```diff
@@ -409,51 +411,51 @@ the configuration takes effect. You can try TestDFSIO for example,
 
 	`file: path matches "/test/*" and accessCount(5m) > 3 | allssd`
 
-This rule means all the files under /test directory, if it is accessed 3 times during
+This rule means for each file under /test directory, if it is accessed 3 times during
 last 5 minutes, SSM should trigger an action to move the file to SSD. Rule engine
-will evaluate the condition every MAX{5s,5m/20} internal.
+will evaluate the condition every MAX{5s,5m/20} interval.
 
 
 ## **Move to Archive(Cold) rule**
 
 	`file: path matches "/test/*" and age > 5h | archive`
 
-This rule means all the files under /test directory, if it's age is more than 5 hours,
+This rule means for each file under /test directory, if its age is more than 5 hours,
 then move the file to archive storage.
 
 ## **Move one type of file to specific storage**
 
 	`file: path matches "/test/*.xml" | allssd`
 
 This rule will move all XML files under /test directory to SSD. In this rule, neither a
-single date nor time value is specified, the rule will be evaluated every short time interval (5s by default).
+single date nor time value is specified, and the rule will be evaluated every short time interval (5s by default).
 
 ## **Specify rule evaluation interval**
 
 	`file: every 3s | path matches "/test/*.xml" | allssd`
   
 This rule will move all XML files under /test directory to SSD. The rule engine will
-evaluate whether the condition meets every 3s. 
+evaluate whether the condition is met every 3s.
 
 
 ## **Backup files between clusters**
      
      `file: every 500ms | path matches "/test-10000-10MB/*"| sync -dest hdfs://sr518:9000/test-10000-10MB/`
 	
-This rule will copy file and update any namespace changes(add,delete,rename,append) under source directory "/test-10000-10MB/" to destination directory "hdfs://sr518:9000/test-10000-10MB/". 
+This rule will copy files and update all namespace changes(add,delete,rename,append) under source directory "/test-10000-10MB/" to destination directory "hdfs://sr518:9000/test-10000-10MB/".
 
 ## **Support action chain**
 
 	`file: path matches "/test/*" and age > 90d | archive ; setReplica 1 `
 	
-SSM use ";" to separate different actions in a rule. The execution trigger of later action depends on the successful execution of the prior action. If prior action fails, the following actions will not be executed.
+SSM use ";" to separate different actions in a rule. Whether the later action will be executed depends on the execution result of the prior action. If prior action fails, the following actions will not be executed.
```

Comment:
In SSM rule, ";" is used to separate two or more actions. These actions are executed in sequence. If a prior action fails, the followed actions will not be executed.